lab.R

```diff
@@ -89,7 +93,9 @@ data("nytcovcounty")
 #' 
 
 #' 1. *Write a pipe that starts with `nytcovcounty` as input, filters down to California and April 1-August 31, 2020, and assigns the result to a variable `filtered_df`.  Hint: You can compare dates as though they were strings, e.g., `date <= '1980-05-17'` gives you dates on or before May 17, 1980.* 
-#' 
+filtered_df= nytcovcounty %>% 
+    filter(date < '2020-08-31' & date >'2020-08-01') %>% 
```

Comment:
This filter will give you August 2 through August 30.  Check the dates and whether you're using strict inequalities.